app/apollo/resolvers/subscription.js

```diff
@@ -793,8 +796,8 @@ const subscriptionResolvers = {
 
         pubSub.channelSubChangedFunc({org_id: org_id}, context);
 
-        // Allow graphQL plugins to retrieve more information. removeSubscription can delete subscriptions. Include details of each deleted resource in pluginContext.
-        context.pluginContext = {subscription: {name: subscription.name, uuid: subscription.uuid}};
+        // Allow graphQL plugins to retrieve more information. removeSubscription deletes a subscription and can delete associated version if specified. Include details of each deleted resource in pluginContext.
+        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid, tags: channel.tags}, subscription: {name: subscription.name, uuid: subscription.uuid, groups: subscription.groups}};
```

Comment:
As noted, deleting a subscription can delete an associated version if specified.  If so, the version details should be included in the plugin context too.